### PR TITLE
handling json data in sql

### LINKED ISSUE #1863: handling json data in sql
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/ajay-dhangar/algo/blob/main/CONTRIBUTING.md).

### Description

handling json data in sql

### Self-service

- [X] I'd be willing to address this documentation request myself.

### PR DESCRIPTION
**Pull request by jainaryan04**
### Description
Handling JSON data within SQL combines the flexibility of JSON with the robustness of SQL databases. By leveraging native JSON support, developers can efficiently store, query, and manipulate JSON data, optimizing database operations for modern applications.

Fixes #1863 

### Type of change
- [X] Documentation update

### Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/languages/SQL/sql-25.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+id: SQL-JSON-Data
```
Comment (ajay-dhangar) [inline docs/languages/SQL/sql-25.md]: replace `id: SQL-JSON-Data` with `id: sql-json-data`

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
update your `id` based on suggestions

**Comment by jainaryan04** [comment]
done @ajay-dhangar

**Review by ajay-dhangar — APPROVED** [review]
Done

**Pull request merged**